java/src/Ice/src/main/java/com/zeroc/Ice/ConnectionI.java

```diff
@@ -1215,21 +1214,13 @@ public synchronized void sendHeartbeat() {
             }
 
             // We send a heartbeat to the peer to generate a "write" on the connection. This
-            // write
-            // in
-            // turns creates a read on the peer, and resets the peer's idle check timer.
-            // When
-            // _sendStream is not empty, there is already an outstanding write, so we don't
-            // need to
-            // send a heartbeat. It's possible the first message of _sendStreams was already
-            // sent
-            // but
-            // not yet removed from _sendStreams: it means the last write occurred very
-            // recently,
-            // which is good enough with respect to the idle check.
+            // write in turns creates a read on the peer, and resets the peer's idle check timer.
+            // When _sendStream is not empty, there is already an outstanding write, so we don't
+            // need to send a heartbeat. It's possible the first message of _sendStreams was
+            // already sent but  not yet removed from _sendStreams: it means the last write
```

Comment:
Extra space hiding here.
```suggestion
            // already sent but not yet removed from _sendStreams: it means the last write
```